static/app/stories/storybook.tsx

```diff
@@ -50,8 +59,16 @@ function Story(props: {name: string; render: StoryRenderFunction}) {
 
   return (
     <Storybook.Section>
-      <Storybook.Title>{props.name}</Storybook.Title>
+      <StoryHeadingContainer>
```

Comment:
All stories now use this heading, and since they all render the table of contents, al titles can now be linked to